flask_nemo/__init__.py

```diff
@@ -237,6 +245,27 @@ def transform(self, work, xml):
         elif func is None:
             return etree.tostring(xml, encoding=str)
 
+    def transform_urn(self, urn):
+        """ Transform urn according to configurable function
+
+        :param urn: URN to transform
+        :type URN
+        :return: String representation of transformed urn
+        :rtype: str
+        """
+        # We check first that we don't have
+        if str(urn) in self.__urntransform:
+            func = self.__urntransform[str(urn)]
```

Comment:
This line is not tested, could you add to the test a situation where this is defined.